docs/getting-started/croeseid-testnet.md

```diff
@@ -184,6 +189,25 @@ Alternatively, you can run it on this [browser based IDE](https://repl.it/@allth
 $ cat ~/.chain-maind_testnet/config/priv_validator_key.json | jq -r '.pub_key.value'
 ```
 
+## Step 4. Transactions subcommands
+### Step 4-1. `tx bank send ` - Transfer operation
+
+Transfer operation involves the transfer of tokens between two addresses.
+
+#### **Send Funds** [`tx bank send <from_key_or_address> <to_address> <amount> <network_id>`]
+
+:::details Example: Send 10cro from a transfer address to another.
+
+```bash
+$ chain-maind tx bank send Default cro17waz6n5a4c4z388rvc40n4c402njfjgqmv0qcp 10cro --chain-id cro-test
```

Comment:
```suggestion
$ chain-maind tx bank send Default 
tcro1j7pej8kplem4wt50p4hfvndhuw5jprxxn5625q 10cro --chain-id cro-test
```
I have just change it to the `tcro...` prefix for this testnet doc, Its all good.